src/contextbuilder.cpp

```diff
@@ -84,6 +84,62 @@ std::vector<std::string> ContextBuilder::listDevices()
 	return uris;
 }
 
+std::string ContextBuilder::getUri(struct iio_context* ctx)
+{
+	if(ctx == nullptr)
```

Comment:
it's a good idea to be consistent with style;
for example: here you are using 
```
if()
{
....
}
```

while in other places, it's 
```
if () {
}
```

whatever the style, it is good to use the one that was used initially when the file was written, and also to keep the same style in a change

Reply:
Now, with commit 5f6c11f, the style is consistent.